SPECS/python3/python3.spec

```diff
@@ -27,11 +28,14 @@ Requires:       openssl
 Requires:       python3-libs = %{version}-%{release}
 Requires:       readline
 Requires:       xz
+Provides:       python
```

Comment:
Could you also add the `= %{version}-%{release}` for this `Provides`? We should be doing that everywhere, but we still have some leftovers.